connectors/salesforce/src/main/resources/application.yml

```diff
@@ -10,6 +10,7 @@ sf:
    opportunityContactLinkPath: services/data/v20.0/sobjects/OpportunityContactRole
    opportunityTaskLinkPath: services/data/v20.0/sobjects/Task
    attachmentTasklinkPath: services/data/v20.0/sobjects/Attachment
+   opportunityFieldsUpdatePath: services/data/v39.0/sobjects/Opportunity/{opportunityId}
```

Comment:
Should we need to update all of them to latest v39.0 ?

Reply:
He created a followup ticket for it: https://jira-euc.eng.vmware.com/jira/browse/APF-1159

Reply:
@shsmysore : Yeah, we should upgrade all the API's to ```v39.0``` if it does not hamper any existing functionality.